style.css

```diff
@@ -103,3 +104,85 @@ p {
     color:#ffffff;
     text-align-last: center;
 }
+/* touch menu*/
+._touch .bottom{
+    position: absolute;
+    bottom: 10px;
+    left: 120px;
+    display: block;
+    width: 0;
+    height: 0;
+    margin: 0 0 0 5px;
+    transition: transform 0.3s ease 0s;
+    border-left: 50px solid transparent;
+    border-right: 50px solid transparent;
+    border-top: 100px solid #fff;
+    z-index: 1;
+    opacity: 20%;
+}
+
+._touch .left{
+    position: absolute;
+    bottom: 10px;
+    left: 10px;
+    display: block;
+    width: 0;
+    height: 0;
+    margin: 0 0 0 5px;
+    transition: transform 0.3s ease 0s;
+    border-bottom: 50px solid transparent;
+    border-right: 100px solid #fff;
+    border-top: 50px solid transparent;
+    z-index: 1;
+    opacity: 20%;
+}
+
+._touch .right{
+    position: absolute;
+    bottom: 10px;
+    left: 230px;
+    display: block;
+    width: 0;
+    height: 0;
+    margin: 0 0 0 5px;
+    transition: transform 0.3s ease 0s;
+    border-bottom: 50px solid transparent;
+    border-left: 100px solid #fff;
+    border-top: 50px solid transparent;
+    z-index: 1;
+    opacity: 20%;
+}
+
+._touch .top{
+    position: absolute;
+    bottom: 120px;
+    left: 120px;
+    display: block;
+    width: 0;
+    height: 0;
+    margin: 0 0 0 5px;
+    transition: transform 0.3s ease 0s;
+    border-left: 50px solid transparent;
+    border-right: 50px solid transparent;
+    border-bottom: 100px solid #fff;
+    z-index: 1;
+    opacity: 20%;
+}
+
+._touch .fire{
+    position: absolute;
+    bottom: 75px;
+    right: 75px;
+    display: block;
+    width: 100px;
+    height: 100px;
+    border-radius: 50%;
+    background-color: #fff;
+    z-index: 1;
+    opacity: 20%;
+}
+
+body ._touch{
+    min-width: 100%;
+    min-height: 100%;
```

Comment:
I think on PC we also have game width and height 100% of screen?

So don't we add it for `body` on all devices, not only for `.touch` ?